include/flatbuffers/flatbuffers.h

```diff
@@ -1470,6 +1502,21 @@ class FlatBufferBuilder
     finished = true;
   }
 
+  // Allocates space for a vector of structures.
+  // Must be completed with EndVectorOfStructs().
+  template<typename T>
```

Comment:
I believe the majority of occurrences of `template` in this code base put them on the same line.